src/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -96,7 +96,7 @@ public static partial class ImmutableArray
     }
     public partial struct ImmutableArray<T> : System.Collections.Generic.ICollection<T>, System.Collections.Generic.IEnumerable<T>, System.Collections.Generic.IList<T>, System.Collections.Generic.IReadOnlyCollection<T>, System.Collections.Generic.IReadOnlyList<T>, System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList, System.Collections.Immutable.IImmutableList<T>, System.Collections.IStructuralComparable, System.Collections.IStructuralEquatable, System.IEquatable<System.Collections.Immutable.ImmutableArray<T>>
     {
-        internal T[] array;
+        private T[] array;
```

Comment:
**Question:** How do we know this is the right change (internal -> private)?

Reply:
It shouldn't make a difference in the struct layout, rather than just not being accessible to anything from within the assembly. cc: @jkotas @ericstj do you guys know if this change is correct?

Reply:
This is fine, as long as it is what GenAPI generates.

Reply:
Since the ref doesn't have an `[InternalsVisibleTo]` there shouldn't be any observable difference in those access levels.